packages/h5wasm/src/h5wasm-api.ts

```diff
@@ -108,13 +108,7 @@ export class H5WasmApi extends ProviderApi {
   private async getH5WasmEntity(path: string): Promise<H5WasmEntity> {
     const file = await this.file;
 
-    const h5wEntity = file.get(path);
-    assertNonNull(
-      h5wEntity,
-      path === '/' ? `Expected valid HDF5 file` : `No entity found at ${path}`
-    );
```

Comment:
Actually, I should perhaps keep this assertion for `path = '/'`

Reply:
> getH5WasmEntity is never called on anything else than a group by the Explorer

Ah, yes indeed, this is the result of our caching strategy. Well, I would prefer we didn't rely on this but this is only until datatypes are not returned as `null`.